_tests/tests_suite_test.go

```diff
@@ -118,24 +121,23 @@ func addKey(name string) {
 	path := path.Join(home, ".ssh", name)
 	// create the key under ~/.ssh/<name> if it doesn't already exist
 	if _, err := os.Stat(path); os.IsNotExist(err) {
-		cmd := "ssh-keygen -q -t rsa -b 4096 -C otto.test@deis.com -f %s -N ''"
-		_, err := execute(cmd, path)
+		cmd := "ssh-keygen -q -t rsa -b 4096 -C %s -f %s -N ''"
+		_, err := execute(cmd, name, path)
 		Expect(err).NotTo(HaveOccurred())
 	}
 	// add the key to ssh-agent
 	_, err := execute("eval $(ssh-agent) && ssh-add %s", path)
 	Expect(err).NotTo(HaveOccurred())
-	// add the public key to deis (assumes the user is logged in)
-	_, err = execute("deis keys:add %s.pub", path)
-	Expect(err).NotTo(HaveOccurred())
 }
 
 func getController() string {
-	host := os.Getenv("DEIS_WORKFLOW_SERVICE_HOST")
+	host := os.Getenv("DEIS_HOST")
```

Comment:
it would be nice to keep this the same as the service env var so that this could easily be run in a pod...